### ISSUE TITLE
version migrations

### ISSUE THREAD

**Issue by kibotu** [tntkhang/realm-encryption-helper#5]
any idea how to handle version migrations with realm encryptions?

**Comment by tntkhang** [comment]
Thanks, I will think about it, do you have any suggestion to contribute?

**Comment by genie137** [comment]
Isnt the migration handles by Realm itself? This libary only provides the encryption key right?

**Comment by tntkhang** [comment]
> Isnt the migration handles by Realm itself? This libary only provides the encryption key right?

Yes, The library only provides the key (generate 64 bit key, save/get it to keystore). The key you use to encrypt the Realm will be store in KeyStore with keyName is your application name.

> any idea how to handle version migrations with realm encryptions?
RealmEncryption version migrations only need to use the same keyName between each version

- The version 1.0.1: will use the name of the application to be the key to save 64bit key in the keystore, it didn't depend on the keyName you input in the initHelper function.
- The version 1.0.2: will use the name are inputed in the initHelper function to be the key to save 64bit key, and remove all debug log.

So basically version 1.0.1 and 1.0.2 are not a big update, it just changes the keyName
